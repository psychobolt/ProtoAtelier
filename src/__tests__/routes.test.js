import React from 'react';
import configureMockStore from 'redux-mock-store';
import { shallow } from 'enzyme';
import createHistory from 'history/createMemoryHistory';

import Routes from '../routes';

const mockStore = configureMockStore([]);

test('component <Routes /> should render correctly', () => {
  const props = {
    history: createHistory({ keyLength: 0 }),
    store: mockStore({}),
  };
  const wrapper = shallow(<Routes {...props} />);
  expect(wrapper).toMatchSnapshot();
});
