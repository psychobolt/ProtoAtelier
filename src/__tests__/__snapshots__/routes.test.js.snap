// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`component <Routes /> should render correctly 1`] = `
<Provider
  store={
    Object {
      "clearActions": [Function],
      "dispatch": [Function],
      "getActions": [Function],
      "getState": [Function],
      "replaceReducer": [Function],
      "subscribe": [Function],
    }
  }
>
  <ConnectedRouter
    history={
      Object {
        "action": "POP",
        "block": [Function],
        "canGo": [Function],
        "createHref": [Function],
        "entries": Array [
          Object {
            "hash": "",
            "pathname": "/",
            "search": "",
            "state": undefined,
          },
        ],
        "go": [Function],
        "goBack": [Function],
        "goForward": [Function],
        "index": 0,
        "length": 1,
        "listen": [Function],
        "location": Object {
          "hash": "",
          "pathname": "/",
          "search": "",
          "state": undefined,
        },
        "push": [Function],
        "replace": [Function],
      }
    }
  >
    <div>
      <Route
        component={[Function]}
        path="/:filter?"
      />
    </div>
  </ConnectedRouter>
</Provider>
`;
